#159051218 Determine color palettes for the app

PromasterGuru opened the pull request:
#### What does this PR do?
- Determines Color Palette for the App

#### Description of Task to be completed?
- Determines the color palettes that will be used in the app
- Updates the README.md to include the color palettes.

#### What are the relevant pivotal tracker stories?
- [#159051218](https://www.pivotaltracker.com/story/show/159051218)

PromasterGuru committed:
feat(color-palettes): Determine color palettes for the app

- Update README.md to add app color palettes

[Delivers #159051218]

joeeasy commented on README.md:
```diff
@@ -1,2 +1,20 @@
+
+There are three wireframes in the wireframe folder displaying both static and interaction elements.
+
+**[Login mockup](https://github.com/PromasterGuru/ConvergeCodelab/blob/ft-design-mockups-158792524/wireframes/Login.png)** : Mocks the user login activity that will be used to authenticate users.
```
Good job on the README, moreover it'll make more sense if there an image.

PromasterGuru replied:
> Good job on the README, moreover it'll make more sense if there an image.

Thank you. I will add the images ASAP.

PromasterGuru committed:
feat(design-mockups): Design Mockups

- Implement mockups for the converge codelab app
- Update README.md with wireframes information.

[Delivers #158792524]
